bioblend/_tests/TestGalaxyInvocations.py

```diff
@@ -29,10 +18,34 @@ def test_invocation(self):
         self.assertEqual(invocation['state'], 'cancelled')
 
         summary = self.gi.invocations.get_invocation_summary(invocation_id)
+        assert summary['states'] == {}
+
+    def test_get_invocation_report(self):
+        invocation = self._invoke_workflow()
+
+        invocation_id = invocation['id']
+        workflow_id = invocation['workflow_id']
         report = self.gi.invocations.get_invocation_report(invocation_id)
+        assert report['workflows'] == {workflow_id: {'name': 'paste_columns'}}
+        with self.assertRaises(Exception):
+            self.gi.invocations.get_invocation_report(invocation_id, format='pdf')
```

Comment:
```suggestion
        try:
            self.gi.invocations.get_invocation_report(invocation_id, format='pdf')
        except Exception:
            # This can fail if dependencies as weasyprint are not installed on the Galaxy server
            pass
```